js/src/layers/AntPath.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+var widgets = require('@jupyter-widgets/base');
+var _ = require('underscore');
+var L = require('../leaflet.js');
+var vectorlayer = require('./VectorLayer.js');
+var LeafletVectorLayerView = vectorlayer.LeafletVectorLayerView;
+var LeafletVectorLayerModel = vectorlayer.LeafletVectorLayerModel;
+var {antPath} = require('leaflet-ant-path');
+
+var LeafletAntPathModel = LeafletVectorLayerModel.extend({
+    defaults: _.extend({}, LeafletVectorLayerModel.prototype.defaults, {
+        _view_name : 'LeafletAntPathView',
+        _model_name : 'LeafletAntPathModel'
```

Comment:
You are missing the model default values (use: 'polyline', radius: 10, etc)

Reply:
Right!